README.md

```diff
@@ -32,13 +32,34 @@ Running in development mode can allow you to run in full node mode instead of th
 
 ### Building the Packaged App
 
-To build the packaged version of the app for your current platform, run:
+To build the packaged version of the app e.g. for macOS run:
 ```
+cp $GOPATH/bin/lnd ./assets/bin/darwin
 npm run electron-pack
 ```
 
 The packaged app will then be available in the lightning-app/dist directory. The packaged version of the app will run on Bitcoin testnet. To debug a packaged app, go to localhost:9997 in your browser.
```

Comment:
Not related to this PR but going to `localhost:9997` doesn't work for me. is this a macOS thing?